src/libraries/Microsoft.PowerFx.Core/Parser/ParserOptions.cs

```diff
@@ -25,6 +25,11 @@ public class ParserOptions
         /// </summary>
         public bool NumberIsFloat { get; set; }
 
+        /// <summary>
+        /// If true, various words have been reserved and are not available for identifiers.
+        /// </summary>
+        public bool ReservedKeywords { get; set; }

```

Comment:
This should not be public on ParserOptions, imo. Everyone other than PA should always get the reserved behavior.

Reply:
probably internal works

Reply:
Done.